components/Row.tsx

```diff
@@ -50,10 +49,11 @@ const collapsibleTheme = createTheme({
       styleOverrides: {
         root: {
           fontWeight: "bold",
-          fontSize: "large",
+          fontSize: "1.1rem",
```

Comment:
It would be better if there was some consistent font sizing, line 35 uses 'px' and here uses 'rem' - unless there is a reason for it?